.circleci/config.yml

```diff
@@ -90,6 +90,9 @@ jobs:
       - checkout
       - install-yarn
       - install-dependencies
+      - run:
```

Comment:
@marmoure do we need to update the `README.md` for prosemirror-lwdita-demo to include this extra build step too?

Reply:
Fixed in [891c14e](https://github.com/evolvedbinary/prosemirror-lwdita/pull/506/commits/891c14e1b2d7c419cbac0a29aac4c587f2fc3913)